File: frameworks/Java/jawn/src/main/java/app/controllers/PlaintextController.java
package app.controllers;

import net.javapla.jawn.core.Controller;

public class PlaintextController extends Controller {

    private static final String message = "Hello, World!";
    private static final byte[] bytemessage = message.getBytes();
    public void index() {
        respond().text(bytemessage);
    }
}

File: frameworks/CSharp/aspnet-mono/src/Controllers/PlaintextController.cs
using System.Text;
using System.Threading.Tasks;
using System.Web.Mvc;

namespace Benchmarks.AspNet.Controllers
{
    public class PlaintextController : Controller
    {
        private static readonly byte[] _helloWorldPayload = Encoding.UTF8.GetBytes("Hello, World!");
        private static readonly string _payloadLength = _helloWorldPayload.ToString();

        public Task Index()
        {
            var response = Response;
            response.StatusCode = 200;
            response.ContentType = "text/plain";
            var payloadLength = _helloWorldPayload.Length;
            response.Headers["Content-Length"] = _payloadLength;
            return response.OutputStream.WriteAsync(_helloWorldPayload, 0, payloadLength);
        }
    }
}